cellprofiler/modules/morphologicalskeleton.py

```diff
@@ -0,0 +1,50 @@
+"""
+
+Morphological skeleton thins an image into a single-pixel wide skeleton.
```

Comment:
Could you add a note that this is intended for use on binary images and label matrices?